packages/create-nx-workspace/src/create-app.ts

```diff
@@ -0,0 +1,79 @@
+import * as ora from 'ora';
+import { join } from 'path';
+import { CreateWorkspaceOptions } from './create-workspace-options';
+import { execAndWait } from './utils/exec-and-wait';
+import { output } from './utils/output';
+import {
+  getPackageManagerCommand,
+  getPackageManagerVersion,
+  PackageManager,
+} from './utils/package-manager';
+import { getFileName, mapErrorToBodyLines } from './utils/string-utils';
+import { unparse } from './utils/unparse';
+
+/**
+ * Create a new Nx workspace
+ * @param tmpDir temporary directory to invoke nx cli
+ * @param name name of new nx workspace
+ * @param packageManager current package manager
+ * @param options options to pass to nx cli
+ * @returns
+ */
+export async function createApp<T extends CreateWorkspaceOptions>(
```

Comment:
Can we rename this to `createEmptyWorkspace`? I was expecting this function's end state to contain an app already, but that's not the case until after `createPreset`.